Test_1/HomeTask_09.py

```diff
@@ -0,0 +1,32 @@
+"""HomeTask_09.
+
+write script that will read "input.txt" file
+and find there all characters from english alphabet
+collect these characters and their positions in file
+after info collected this info as text write
+to "output.txt" file in such format
+####################
+<first found letter> -> pos1
+<next_letter> -> pos2
+<next_letter -> pos3
+etc
+#######################
+"""
+import re
+
+fh = open('input.txt', 'rt')
+input_text = fh.read()
+fh.close()
+
+english_letter = []
+for pos, el in enumerate(input_text, start=1):
+    if bool(re.search('[a-zA-Z]', el)):
+        english_letter.append((el, str(pos)))
```

Comment:
```suggestion
english_letter = [(el, str(pos))
                  for pos, el in enumerate(input_text, start=1)
                  if re.search('[a-zA-Z]', el)]
```

Better organize  such patterns as list comprehensions. Also do not forget that "if" will do bool for you automatically